enhancements/ingress/lb-subnet-selection-aws.md

```diff
@@ -0,0 +1,597 @@
+
+While the AWS Cloud Controller Manager (CCM) handles invalid annotations by
+producing errors and events, it is difficult for the Ingress Operator to
+decipher these CCM events _after_ the load balancer has been provisioned.
+Therefore, the Ingress Operator can't indicate to a cluster admin that an
+IngressController's load balancer is in a malfunctioning state (i.e., the
+service is not getting reconciled) while the service is already provisioned.
+
+When the IngressController is first created and the LoadBalancer-type
+service is not yet provisioned, these same invalid `Subnets` values will
+prevent the load balancer from being provisioned. The existing Ingress
+Operator logic will clearly indicate to the cluster admin that the load balancer
+failed to provision via the `LoadBalancerReady` status condition. In addition,
+the `LoadBalancerReady` condition will include the CCM event logs that indicate
+to the cluster admin that the `Subnets` value is invalid (see [Support Procedures](#support-procedures)
+for examples). Therefore, requiring manual service deletion provides a way
+for the Ingress Operator to produce a clear signal to the cluster admin
+that `Subnets` is invalid.
+
+See [Invalid Subnet Annotation Values](#invalid-subnet-annotation-values) for
+examples of invalid subnet annotation values.
+
+##### Reason #2: To mitigate upgrade compatibility issues.
+
+**Note**: Directly configuring the `service.beta.kubernetes.io/aws-load-balancer-subnets`
+on the LoadBalancer-type service is not supported and has never
+been. However, this enhancement is designed to prevent cluster disruption upon
+upgrading with this unsupported configuration.
+
+If the `service.beta.kubernetes.io/aws-load-balancer-subnets` annotation was
+previously configured and the cluster is upgraded, the default value `[]` for
+`Subnets` would clear the annotation for existing IngressControllers, which
+would break cluster ingress. However, requiring intervention to effectuate
+a `spec.endpointPublishingStrategy.loadBalancer.providerParameters.aws.subnets`
+value different from the current value of the annotation prevents this
+automatic removal of the service annotation on upgrade.
+
+**Note**: Cluster admins upgrading with an unmanaged subnet annotation don't
+need to delete and recreate the service, instead they can follow the
+[Unmanaged Subnet Annotation Migration Workflow](#unmanaged-subnet-annotation-migration-workflow).
+
+##### Reason #3: The CCM Doesn't Reconcile NLB Subnets Updates after creation.
+
+The CCM doesn't reconcile `service.beta.kubernetes.io/aws-load-balancer-subnets`
+annotation updates to NLBs once it has been created. However, requiring a service
+deletion and recreation will effectuate these changes.
+
+See [CCM Doesn't Reconcile NLB Subnets Updates](#ccm-doesnt-reconcile-nlb-subnets-updates)
+for more details on why the CCM doesn't reconcile these updates.
+
+### Install Config Proposal (Day 1)
+
+TODO: Install time configuration proposal
```

Comment:
I strongly believe that you should remove the Day 1 references from all the places it shows in the EP and mention Day 1 in Open Questions, because it is an open item.